PURL-SPECIFICATION.rst

```diff
@@ -136,6 +136,8 @@ The rules for each component are:
   - The ``type`` MUST start with an ASCII letter.
   - The ``type`` MUST NOT be percent-encoded.
   - The ``type`` is case insensitive. The canonical form is lowercase.
+  - ``purl`` parsers MUST return an error when the ``type`` contains a prohibited
```

Comment:
this change is not in the scope of the https://github.com/package-url/purl-spec/issues/382


Anyway, :+1: this is in accordance with the existing spec.

https://github.com/package-url/purl-spec/blob/8040ff0be50f0c5b1986b1a0947bd539f5405fc4/PURL-SPECIFICATION.rst?plain=1#L245

Reply:
Thanks @jkowalleck -- I forgot my own advice to keep PRs narrowly focused.  Will remove in the next iteration and address in a separate PR.  :+1:

Reply:
Backed out the type update.